packages/config/src/handlers/dashboardQueries/defaultQueries.ts

```diff
@@ -616,7 +616,8 @@ const declarations = ({ lastUpdatedAt }: { lastUpdatedAt: string }) => ({
     { $unwind: '$state' },
     {
       $project: {
-        _id: 1,
+        _id: 0,
```

Comment:
Removing the `_id` field from being projected, as we want to create a new one for every document. And as we are using the new `id` field for deduping, it doesn't matter if a new one is created